packages/flutter/lib/src/rendering/child_view.dart

```diff
@@ -10,13 +11,28 @@ import 'package:mojo_services/mojo/ui/layouts.mojom.dart' as mojom;
 import 'package:mojo_services/mojo/ui/view_provider.mojom.dart' as mojom;
 import 'package:mojo_services/mojo/ui/views.mojom.dart' as mojom;
 import 'package:mojo/application.dart';
+import 'package:mojo/core.dart' as core;
 import 'package:mojo/mojo/service_provider.mojom.dart' as mojom;
 
 import 'box.dart';
 import 'object.dart';
 
-// TODO(abarth): Populate the view host.
-final mojom.ViewHost _viewHost = null;
+mojom.ViewHostProxy _initViewHostProxy() {
+  try {
+    return new mojom.ViewHostProxy.fromHandle(
+      new core.MojoHandle(internals.takeViewHostHandle()));
+  } catch (e) {
+    return null;
```

Comment:
I'm not sure what the right error handling is here.  The reality is that the callers aren't actually prepared to get a null handle out, so something terrible is going to happen pretty soon.  We might want to skip the try/catch entirely.